Library/Homebrew/dev-cmd/release-notes.rb

```diff
@@ -29,6 +33,15 @@ def release_notes
     args = release_notes_args.parse
 
     previous_tag = args.named.first
+
+    if previous_tag.present?
+
+      previous_tag_date = Date.parse Utils.popen_read(
+        "git", "-C", HOMEBREW_REPOSITORY, "log", "-1", "--format=%aI", previous_tag.sub(/\d+$/, "0")
+      )
+      opoo "The latest major/minor release was less than one month ago." if previous_tag_date > (Date.today << 1)
```

Comment:
What is `(Date.today << 1)` doing? A variable would also make this more obvious, I think. `Date.today - 1.month` work?

Reply:
Doesn't work:

```
Error: undefined method `month' for 1:Integer
```

I'll extract to a variable `one_month_ago`